sys/src/9/riscv/ether.c

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
I'd be happier not to have this. Another option is to put an
ethernet.json
in port/ and only have amd64 use it.